workflows/models.py

```diff
@@ -653,8 +638,12 @@ def get_lanes(self):
         return lanes
 
     def create_output_datasets(self, tag_name=None, update=False):
-        """
-        """
+        storage_client = tantalus_api.get_storage_client(
+            self.storages["working_results"])
+        metadata_yaml_path = os.path.join(
+            self.jira, "results", self.analysis_type, "metadata.yaml")
+        metadata_yaml = yaml.safe_load(storage_client.read(metadata_yaml_path))
```

Comment:
this should work: `yaml.load(storage_client.open_file(metadata_yaml_path))`, then you dont need the change to `tantalus.py`